app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(team: list) -> None:
+    total_rating = 0
+    for player in team:
+        total_rating += player.get_rating()
+    return total_rating
+
+
+def elves_concert(elves: list) -> None:
```

Comment:
It would be better if you specify the type annotation for the list items, because this function can only work for objects that have the "play_elf_song" method.
Do the same for Dwarfs